tests/functional/c/consider/consider_using_dict_items.py

```diff
@@ -1,5 +1,6 @@
 """Emit a message for iteration through dict keys and subscripting dict with key."""
 # pylint: disable=line-too-long,missing-docstring,unsubscriptable-object,too-few-public-methods,redefined-outer-name,use-dict-literal,modified-iterating-dict
+# pylint: disable=wrong-import-position

```

Comment:
```suggestion
# pylint: disable=wrong-import-position

import os

```
Might as well import at the top of the file and apply black if we add a line and already must do a full output regeneration because the line changed.